users/views.py

```diff
@@ -68,4 +68,51 @@ def post(self, request):
             return JsonResponse({"MESSAGE":"INVALID_USER"}, status=401)
 
         except KeyError:
-            return JsonResponse({"MESSAGE":"KEY_ERROR"}, status=400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE":"KEY_ERROR"}, status=400)
+
+class UserDetailView(View):
+    @login_decorator
+    def get(self, request):
+        user = User.objects.get(id = request.user.id)
+        result = {
+            "name"           : user.name,
+            "email"          : user.email,
+            "is_sms_agree"   : user.is_sms_agree,
+            "is_email_agree" : user.is_email_agree,
+        }
+        return JsonResponse({"Result": result}, status=200)
```

Comment:
이미 request 객체에 user가 담겨있기 때문에 또 다시 user를 호출할 필요가 없을 것 같네요~
```suggestion
        user = request.user
        
        result = {
            "name"           : user.name,
            "email"          : user.email,
            "is_sms_agree"   : user.is_sms_agree,
            "is_email_agree" : user.is_email_agree,
        }
        
        return JsonResponse({"Result": result}, status=200)
```

Reply:
놓친부분이었습니다!! 감사합니다!